.orchestra/support/revng-container/apt.no-extra

```diff
@@ -0,0 +1,4 @@
+APT::Get::Install-Recommends "false";
+APT::Get::Install-Suggests "false";
+APT::Install-Suggests "0";
+APT::Install-Recommends "0";
```

Comment:
Missing blank line, in other places too.

Reply:
Should I `revng-check-convention` the rest of orchesta? there's a lot of stuff that's not passing
(Also there's the catch22 problem of checking orchestra with rcc when orchestra has to install rcc)

Reply:
Nah.